File: TFSWayUI/src/app/promoter/promoter.ts
export interface IPromoter {
    promoterId: number;
    companyId: number;
    projectId: number;
    name: string;
    changedName: string;
    changedNameDate: string;
    cin: string;
    officeAddress: string;
    pan: string;
    incorporationDate: string;
    chkPromoter?: boolean;
    isMainPromoter: string;
    createdDate: string;
    createdBy: string;
    updatedDate: string;
    updatedBy: string;
}

File: TFSWay.DomainModel/Entities/Promoter.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations;

namespace TFSWay.DomainModel.Entities
{
  public  class Promoter
    {
        [Key]
        public int PromoterId { get; set; }
        public int CompanyId { get; set; }
        public int ProjectId { get; set; }
        public string  Name{ get; set; }
        public string  ChangedName{ get; set; }
        public DateTime? ChangedNameDate { get; set; }
        public string  CIN{ get; set; }
        public string  OfficeAddress{ get; set; }
        public string  PAN{ get; set; }
        public DateTime? IncorporationDate { get; set; }
        public string IsMainPromoter { get; set; }
        public DateTime? CreatedDate { get; set; }
        public string CreatedBy { get; set; }
        public DateTime? UpdatedDate { get; set; }
        public string UpdatedBy { get; set; }

    }
}
